src/tests/cli_tests.py

```diff
@@ -841,9 +841,6 @@ def rnp_check_features():
     RNP_BRAINPOOL = re.match(r'(?s)^.*Curves:.*brainpoolP256r1.*brainpoolP384r1.*brainpoolP512r1.*', out)
 
     RNP_IDEA = re.match(r'(?s)^.*Encryption:.*IDEA.*', out)
-    # Check that everything is enabled for Botan:
```

Comment:
Just wanted to check why we had this before and whether the intention is still present today. Thanks!